samples/suit/smp_transfer/README.rst

```diff
@@ -0,0 +1,392 @@
+:orphan:
+
+.. _nrf54h_suit_sample:
+
+SUIT DFU: Firmware update on the nRF54H20 SoC
+#############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The sample demonstrates how to update and boot the nRF54H20 System-on-Chip (SoC) using the Software Update for Internet of Things (SUIT) procedure on the Application and Radio Cores of the SoC.
+The update on the nRF54H20 SoC can be done over Bluetooth® Low Energy or UART.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf54h20dk_nrf54h20_cpuapp
+
+You need the nRF Device Manager app for SUIT update over Bluetooth Low Energy:
+
+* `nRF Device Manager mobile app for Android`_
+  (The minimum required version is v1.9.)
+
+* `nRF Device Manager mobile app for iOS`_
+  (The minimum required version is v1.5.)
+
+For a SUIT update over UART, you need to install MCUmgr.
+MCUmgr is a tool that can be used to upload SUIT envelopes through the SMP protocol.
+For more information on MCUmgr installation, see :ref:`zephyr:mcu_mgr`.
```

Comment:
Does this mean that one also needs to (or can) use the samples/common MCUmgr component if I want to use MCUmgr?

Reply:
@tomchy I believe Greg maybe referring to the `CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` Kconfig option found in samples/common (See [here](https://github.com/nrfconnect/sdk-nrf/pull/15166/files#diff-73049ba9c9a3b317f962ed3fbce65d49a907d8d57a56946e0ac6185fce2d93feR167-R169) for description).

My guess is no, it is not required and/or it has not been tested for SUIT specifically yet?

Reply:
Yes, I'm referring to `CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU`. When we discussed `CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` in the call, Kamil Piszczek said it can be used with whatever sample that uses Bluetooth LE, so I'd like to make sure there are no collisions between that Kconfig option ("MCUmgr component") and this implementation.

Reply:
It cannot be used ATM, because it depends on `BOOTLOADER_MCUBOOT` which is not compatible with nRF54Hxx devices.
I have not heard of this Kconfig before and I do not know if it should be supported on nRF54H (so it would depend on the SUIT and our implementation of SMP image group).